eva/executor/load_multimedia_executor.py

```diff
@@ -122,3 +131,17 @@ def _rollback_load(
     ):
         if do_create:
             storage_engine.drop(table_obj)
+
+    def _is_media_valid(
+        self,
+        file_path: Path,
+    ):
+        file_path = Path(file_path)
+        if validate_media(file_path, self.media_type):
+            return True
+        else:
+            err_msg = f"Load {self.media_type.name} failed due to invalid file {str(file_path)}"
+            # If image is corrupted, do not abord the entire loading process.
+            logger.critical(err_msg)
+            # raise ValueError(file_path)
```

Comment:
@gaurav274 Thought on this? Should we just abort as what we did previously? Since, the media validation process is parallelized, so the abort happens much quickly. Or do we just silently pass those corrupted images but give a warning to users?

Reply:
Will it be possible to return the files that we couldn't load?

Reply:
We can return corrupted files, but then we need to raise exception after we process all files. Should I add this feature in this PR?

2 option sounds good to me. I think we can merge this PR for now by sticking to the original approach: raise exception and abort. I can file another PR to support 2 options.

Reply:
Sounds good!

Reply:
We can fix the build and merge this.